client/state/themes/selectors.js

```diff
@@ -729,8 +729,7 @@ export function getJetpackUpgradeUrlIfPremiumTheme( state, themeId, siteId ) {
 
 /**
  * Returns the price string to display for a given theme on a given site:
- *
- * TODO: Add tests!
+ * `@TODO Add tests!`
```

Comment:
Is this you? Do we need to add the tests then?

Reply:
I added in order to make the prettier pass.